src/FixedDataTable.js

```diff
@@ -602,12 +611,12 @@ var FixedDataTable = React.createClass({
         />;
     }
 
-    if (
-      (state.ownerHeight != null &&
-        state.ownerHeight < state.height &&
-        state.scrollContentHeight + state.reservedHeight > state.ownerHeight) ||
-      state.scrollY < maxScrollY
-    ) {
+    // ownerScrollAvailable is true if there's more room to scroll after
```

Comment:
Thanks, changed it to, "ownerScrollAvailable is true if the rows rendered will overflow the owner element so we show a shadow in that case even if the FDT component can't scroll anymore"